internal/mailer/mailer.go

```diff
@@ -136,13 +156,16 @@ func (m *Mailer) sendWithAuth(
 	}
 	resultChan := make(chan result, 1)
 
+	to = append(to, cc...)
+	to = append(to, bcc...)
+
```

Comment:
It seems that it merges `cc` and `bcc` into `to` addresses. Could you check if this is correct implementation?